packages/neo-one-node-bin/src/utils/createMonitor.ts

```diff
@@ -0,0 +1,68 @@
+import { DefaultMonitor, Monitor } from '@neo-one/monitor';
+import { createLogger, format, transports } from 'winston';
+// hookup config$ stuff?
+
+export const createMonitor = ({ logConsole }: { readonly logConsole?: boolean }): Monitor => {
+  const formats = format.combine(format.timestamp(), format.json());
+  const logger = createLogger({
+    transports: logConsole
+      ? [
+          new transports.Console({
+            level: 'silly',
```

Comment:
Probably want to set this to something higher by default (and make it configurable with rc later)